src/lightgbm/src/main/scala/TrainUtils.scala

```diff
@@ -363,4 +386,41 @@ private object TrainUtils extends Serializable {
     }
   }
 
+  def trainLightGBMBarrier(networkParams: NetworkParams, labelColumn: String, featuresColumn: String,
```

Comment:
Is this supposed to be used up by my other comment? I think you didn't use it up there

Reply:
woops, this was written during an earlier refactor and I forgot to remove it, removed!